core/src/apps/homescreen/__init__.py

```diff
@@ -91,7 +92,7 @@ def get_features() -> Features:
 
 
 async def handle_ClearSession(ctx: wire.Context, msg: ClearSession) -> Success:
-    cache.clear(keep_passphrase=True)
```

Comment:
why did ClearSession keep the passphrase in the past?

Reply:
Let's ask @prusnak if he remembers, I don't recall.

Reply:
Usability